File: app/src/main/java/farasense/mobile/view/components/FaraSenseEditText.java
package farasense.mobile.view.components;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;
import android.view.KeyEvent;
import android.widget.EditText;

public class FaraSenseEditText extends EditText {

    public FaraSenseEditText(Context context, AttributeSet attrs) {
        super(context, attrs);
        this.setTypeface(Typeface.createFromAsset(context.getAssets(), "fonts/Montserrat-Regular.ttf"));
    }
}

File: app/src/main/java/old_farasense/view/components/FaraSenseEditText.kt
package old_farasense.view.components

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import android.widget.EditText

class FaraSenseEditText(context: Context, attrs: AttributeSet) : EditText(context, attrs) {
    init {
        this.typeface = Typeface.createFromAsset(context.assets, "fonts/Montserrat-Regular.ttf")
    }
}